### PR TITLE
C2 mirna yacout c4 f5 record approval of a synonym

### PR DESCRIPTION
**Pull request by Mirna-Yacout**
@AmrARaouf @karimngr @Omar-Hossam plz review

### CODE REVIEW

**Comment by Mirna-Yacout** [comment]
@m0hamed

**Comment by m0hamed** [comment]
what is the issue number for this

**Comment by Mirna-Yacout** [comment]
issue #81

**Comment by ghost** [comment]
@Mirna-Yacout  i think @m0hamed  had some comments on the issue about things you need to change commit the changes then tag us to review

**Comment by ghost** [comment]
here are the comments : please move your method to after the validations
rename the variables from sid to something more descriptive
no brackets are required after the if statement

**Comment by Mirna-Yacout** [comment]
@m0hamed plz review

File: arability/app/models/synonym.rb [hunk 1]
```diff
@@ -14,19 +16,15 @@ class Synonym < ActiveRecord::Base
   #  returns false if the synonym doesnot exist in the database
   #  or if the approval failed to be saved in the database 
   class << self
```
Comment (m0hamed) [inline arability/app/models/synonym.rb]: please use the same class self as the rest of the methods

**Comment by m0hamed** [comment]
when you update please change the issue labels

File: arability/app/models/synonym.rb [hunk 2]
```diff
@@ -13,22 +17,16 @@ class Synonym < ActiveRecord::Base
...
       return false
     end
-  end
-end
-
-  has_many :votes
-  validates_format_of :name, :with => /^([\u0621-\u0652 ])+$/, :on => :create, :message => "The synonym is not in the correct form"
-
-  class << self
+
```
Comment (m0hamed) [inline arability/app/models/synonym.rb]: remove extra empty line

File: arability/app/models/keyword.rb [hunk 3]
```diff
@@ -21,6 +21,7 @@ def approve_keyword(kid)
       end
       return false
     end
+  end
```
Comment (m0hamed) [inline arability/app/models/keyword.rb]: this should be fixed in the other issue

**Comment by Mirna-Yacout** [comment]
@m0hamed done

File: arability/app/models/keyword.rb (hunk 3 above)
Comment (m0hamed) [inline arability/app/models/keyword.rb]: please leave this to the other issue

**Comment by Mirna-Yacout** [comment]
@m0hamed ?

**Comment by m0hamed** [comment]
works for me :+1:

**Comment by Mirna-Yacout** [comment]
@karimngr would u please review and if correct verify and merge?

**Comment by ghost** [comment]
approved

**Pull request merged**